AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm/README.md

```diff
@@ -0,0 +1,124 @@
+# `Genetic Algorithms on GPU using Intel® Distribution of Python numba-dpex` Sample
+
+The `Genetic Algorithms on GPU using Intel® Distribution of Python numba-dpex` sample shows how to implement a general generic algorithm (GA) and offload computation to GPU using numba-dpex.
+
+| Area                    | Description
+| :---                    | :---
+| What you will learn     | How to implement the genetic algorithm using the Data-parallel Extension for Numba* (numba-dpex)?
+| Time to complete        | 8 minutes
+| Category                | Code Optimization
+
+>**Note**: The libraries used in this sample are available in Intel® Distribution for Python* as part of the [Intel® AI Analytics Toolkit (AI Kit)](https://software.intel.com/en-us/oneapi/ai-kit).
+
+## Purpose
+
+In this sample, you will create and run the general genetic algorithm and optimize it to run on GPU using Intel® Distribution for Python* numba-dpex. You will learn what are selection, crossover and mutation, and how to adjust those methods from general genetic algorithm to a specific optimization problem which is the Traveling Salesman Problem.
+
+## Prerequisites
+
+| Optimized for           | Description
+|:---                     |:---
+| OS                      | Ubuntu* 20.04
+| Hardware                | GPU
+| Software                | Intel® AI Analytics Toolkit (AI Kit)
+
+### For Local Development Environments
+
+You will need to download and install the following toolkits, tools, and components to use the sample.
+
+- **Intel® AI Analytics Toolkit (AI Kit)**
+
+  You can get the AI Kit from [Intel® AI Analytics Toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit-download.html).
```

Comment:
I'm not sure if it is right to point user to AI Tools for this specific sample. IDP is not included into AI Tools, only some packages are included, and we don't mention IDP on AI Tools pages. Maybe we need to point to standalone IDP instead?

Another thing - the overall README needs DX review to ensure instructions are up-to-date. We have a README template that can help with that, but for this specific sample some adjustments may be needed. There should also be a trademarks review by InfoDev.

Let's discuss more via e-mail and we can also schedule a short meeting